cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/utils.go

```diff
@@ -0,0 +1,299 @@
+/*
+Copyright © 2021 VMware
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"math"
+	"strings"
+
+	"github.com/kubeapps/common/datastore"
+	"github.com/kubeapps/kubeapps/pkg/chart/models"
+	"github.com/kubeapps/kubeapps/pkg/dbutils"
+	_ "github.com/lib/pq"
+	log "k8s.io/klog/v2"
+)
+
+// TODO(agamez): logic temporaryly extracted from:
+// /cmd/assetsvc/postgresql_utils.go
+// /cmd/assetsvc/utils.go
+// we can either: export this logic under pkg or re-implement the data access layer with an ORM
```

Comment:
Why not ensure this functionality is exported in its current location? If that's an issue, keen to know why. If it's not an issue, I'd recommend just doing it as a separate, pre-quel branch, that is:

-  Public interface `assetManager` -> `AssetManager`
-  Public NewPGManager
-  Public methods

In fact, let me see if I can do that for you to save you the time: #3036 (if you agree, just land that and update).

Reply:
Thank you for performing that changes. I initially opted to copy-n-paste the required logic just to have a clear idea of what was really necessary and what's not. 
Also, I didn't want to mix things up, so I just thought, ok let's do it in another PR, but I didn't manage to do it yet, haha.

However, after yours, I've created this another PR https://github.com/kubeapps/kubeapps/pull/3040. More details there, but I had to move the files to a different folder and go package so that I was able to import them. Perhaps I'm missing something.